scripts/aws/setupdb.sh

```diff
@@ -44,6 +45,8 @@ while getopts ":hbsdpmqf:" opt; do
             load_water_quality=true ;;
         f)
             file_to_load=$OPTARG ;;
+        x)
```

Comment:
You're right, I had an amended commit that I failed to push up.  Updated with newer commit with `getopts`.